packages/user-service/.env.example

```diff
@@ -14,8 +14,12 @@ DB_NAME=db_name
 # DB_USER=
 # DB_PASSWORD=
 
+## Certificate Configuration
+SSL_KEY=/etc/pki/tls/private/server.key
+SSL_CERT=/etc/pki/tls/certs/server.crt
+
 ## Cookie Parser
-COOKIE_PARSER=LDAP_SECRET
+COOKIE_PARSER=USER_SECRET
```

Comment:
You don't have to add any values to the example file. Leave this empty.

Reply:
Updated